packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -130,7 +131,7 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
               transition: `left ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction1}`
             },
             props.disabled && {
-              borderColor: theme.palette.neutralTertiaryAlt,
+              borderColor: semanticColors.buttonBackgroundChecked, // None of the disabled semantic slots had a value of neturalTertiaryAlt
```

Comment:
See the first comment. This should be some disabled color slot. Even if we need to add a new one.